Dockerfile

```diff
@@ -1,7 +1,7 @@
 # ---
 # Stage 1: Install certs, build binary, create default config file
 # ---
-FROM docker.io/golang:1.13.6-alpine3.11 AS builder
+FROM docker.io/golang:1.15.3-alpine3.12 AS builder
```

Comment:
Currently go.mod, bazel and .travis uses go 1.14, I think we should first verify on 1.15 before changing it here.